Examples.md

```diff
@@ -29,7 +31,20 @@ async with grpc.insecure_channel('localhost:50051') as channel:
     response = await stub.Hi(...)
 ```
 
-### Ideal
+### Option 3
+
+```Python
+ctx = grpc.Context() \
+    .with_timeout_secs(5.0) \
```

Comment:
Don't have a strong preference here, though `datetime.timedelta` is a bit more verbose.
Also `datetime.datetime` itself is harder to use for "deadline" side of the API, so it will make API non-symmetric for timeouts and deadlines.